packages/files-ui/src/Components/Elements/Notifications/NotificationsDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react"
+import { Button, BellIcon, MenuDropdown } from "@chainsafe/common-components"
+import { createStyles, ITheme, makeStyles } from "@chainsafe/common-theme"
+import NotificationList from "./NotificationList"
+
+const useStyles = makeStyles(({ palette, constants }: ITheme) =>
+  createStyles({
+    notificationsButton: {
+      position: "relative",
+      "span": {
+        transition: "none"
+      }
+    },
+    badge: {
+      position: "absolute",
+      background: palette.additional["volcano"][6],
+      color: palette.additional["gray"][1],
+      top: "-2px",
+      left: "13px",
+      borderRadius: constants.generalUnit,
+      padding: `${constants.generalUnit * 0.25}px ${constants.generalUnit * 0.5}px`,
+      fontSize: "11px",
+      lineHeight: "11px",
+      height: "0.9rem",
+      minWidth: "1rem"
+    },
+    icon: {
+      transition: "none"
+    }
+  })
+)
+
+export interface Notification {
+  title: string
+  createdAt: Date
```

Comment:
Rather use a number (unix timestamp) here, as it is easier to sort and work with.
```suggestion
  createdAt: number
```